### DISCUSSION TITLE
Set defaut 'DirPort' to '80' for all Exits

### DISCUSSION THREAD

**Discussion by Le1b1** [nusenu/ansible-relayor#230] [Ideas]
... so that all exits will have a exit notice on the http port by default.
Maybe set the DirPort for the second instance to '0' or '8080', because since tor [0.4.6.5](https://blog.torproject.org/new-stable-security-releases-03515-0449-0459-0465/) DirPorts are only used on authorities, relays no longer advertise them.

**Comment by nusenu** [comment]
I see your reasoning but
I'm hesitant to change such fundamental defaults
because it might break **existing** setups when upgrading
the used relayor version because they use that port for something else.
Breaking changes should have important reasons
and since it is so easy to change dirports in tor_ports
I'm unlikely to change the defaults here.

At the time I did explicitly not choose 80 as default,
to avoid conflicting with webservers running on the same
IP address.

kind regards,
nusenu
